doc/changelog/02-specification-language/19519-coercion-field-cleanup.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  ``:>`` now consistently declares coercions in all records (including
```

Comment:
Can you explain what you mean by "consistently"?  Even reading the change log you referred to, it's not clear.  Something like "before :> did XXX, now it does YYY".  And drop the word "consistently"--it tells you very little.

Also, shouldn't there be some change to the description of `:>` in the documentation?  I see you only removed the warnings from the one chapter.

Reply:
> Can you explain what you mean by "consistently"? Even reading the change log you referred to, it's not clear. Something like "before :> did XXX, now it does YYY". And drop the word "consistently"--it tells you very little.

Done

> Also, shouldn't there be some change to the description of `:>` in the documentation? I see you only removed the warnings from the one chapter.

Yes and since this is the last stone of a long chain of deprecations, those changes were done long time ago (and you probably already reviewed them ;-) )